### DISCUSSION TITLE
use external package

### DISCUSSION THREAD

**Discussion by ahmed-abess** [directus/directus#13636] [Archive]
hello 

can i add an extarnal package like f
import FullCalendar from "@fullcalendar/vue";
import dayGridPlugin from "@fullcalendar/daygrid";
import timeGridPlugin from "@fullcalendar/timegrid";
import interactionPlugin from "@fullcalendar/interaction";
import bootstrapPlugin from "@fullcalendar/bootstrap";
import listPlugin from "@fullcalendar/list";
and use it in module  ??

**Comment by rijkvanzanten** [comment]
Yup!

**Reply by ahmed-abess** [reply]
haw ?? there is an exemple plz

**Reply by ahmed-abess** [reply]
should i install it in directus node-module or module node-module ??

**Reply by rijkvanzanten** [reply]
Assuming you're using the directus-extensions package, the app extensions have a bundler that will pull in the node_modules, so you can use vue plugins as you normally would 👍🏻

**Reply by ahmed-abess** [reply]
sure i m using  directus-extensions

**Reply by ahmed-abess** [reply]
when i import calender , i get not found page for module ??
